styles.css

```diff
@@ -345,6 +345,16 @@ body {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+    z-index: 2;
+  }
+
+  .gray-1 {
```

Comment:
I used z-index to make this element appear in front of the others by setting it to an index of 1 and purple-4 and menu-body-mobile to 2. The higher the number the closer to the "front" of the page the element will be.